src/common/components/footer/icon-social.svg

```diff
@@ -1,3 +1,33 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="90px" width="180px" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 180 90"><!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
```

Comment:
If the file is on assets server, I guess it would be fine to link to it directly and not host on BSI.

On the other hand, it seems like this SVG contains some metadata, so maybe it would be worth shrinking it with something like https://jakearchibald.github.io/svgomg/